cc/main.cc

```diff
@@ -145,6 +151,17 @@ std::string GetOutputName(absl::Time now, size_t i) {
   return absl::StrCat(timestamp, "-", hostname, "-", i);
 }
 
+std::string GetRowPrefix(absl::Time now, int id) {
+  std::ostringstream out;
+  absl::TimeZone utc = absl::UTCTimeZone();
+  out << absl::FormatTime("%Y%m%d-%H%M%S", now, utc)
```

Comment:
Is the unique ID for a game the combination of timestamp & thread ID? If so, we should probably use more fine grained granularity than one second. It's entirely possible (though admittedly unlikely) for a game to take less than one second with a low number of reads. For example, randomization during selfplay could cause black to open on a corner point, after which black will resign on its next turn (once training has progressed sufficiently).

Reply:
At present, the unique ID is definitely timestamp + thread ID, so the point you bring up is an important one. I could add more precision to the time stamp, but based on a discussion today with amj and sethtroisi, a timestamp is less useful than a true (and atomic) game ID.

I may do the work to move to an atomic game ID in a later PR, in the interest of keeping this one manageable in size.